extensions/jaxb/deployment/src/main/java/io/quarkus/jaxb/deployment/JaxbProcessor.java

```diff
@@ -275,8 +292,35 @@ void registerClasses(
                 .produce(new ServiceProviderBuildItem(JAXBContext.class.getName(), "com.sun.xml.bind.v2.ContextFactory"));
     }
 
+    @BuildStep
+    @Record(ExecutionTime.STATIC_INIT)
+    void setupJaxbContextConfig(Capabilities capabilities,
+            List<JaxbClassesToBeBoundBuildItem> classesToBeBoundBuildItems,
+            JaxbContextConfigRecorder jaxbContextConfig) {
+        for (JaxbClassesToBeBoundBuildItem classesToBeBoundBuildItem : classesToBeBoundBuildItems) {
+            jaxbContextConfig.addClassesToBeBound(classesToBeBoundBuildItem.getClasses());
+        }
+
+        if (capabilities.isPresent(Capability.HIBERNATE_VALIDATOR)) {
+            Class<?> validationReportClass;
+            if (capabilities.isPresent(Capability.RESTEASY)) {
+                validationReportClass = org.jboss.resteasy.api.validation.ViolationReport.class;
```

Comment:
This should be produced in the `RestEasy` extension

Reply:
Moved to `resteasy-jaxb`